model.py

```diff
@@ -11,23 +12,28 @@
 
 
 class Challenge:
-    def __init__(self, c_info: Dict[str, Any], user_profile: Dict[str, Any]) -> None:
-        self.id = c_info["id"]
-        self.rated = c_info["rated"]
-        self.variant = c_info["variant"]["key"]
-        self.perf_name = c_info["perf"]["name"]
-        self.speed = c_info["speed"]
-        self.increment: int = c_info.get("timeControl", {}).get("increment")
-        self.base: int = c_info.get("timeControl", {}).get("limit")
-        self.days: int = c_info.get("timeControl", {}).get("daysPerTurn")
-        self.challenger = Player(c_info.get("challenger") or {})
-        self.opponent = Player(c_info.get("destUser") or {})
+    """Store information about a challenge."""
+
+    def __init__(self, challenge_info: Dict[str, Any], user_profile: Dict[str, Any]) -> None:
+        """:param user_profile: Information about our bot."""
```

Comment:
Need docstring on parameter `challenge_info`. Other methods in the class are missing `param` docstrings.